addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -70,6 +70,7 @@ $spacers: (
 // grep: BOXED_BODY_BG_ODOO
 $body-bg: o-color('o-cc1-bg') !default;
 $body-color: o-color('o-cc1-text') or color-contrast($body-bg) !default;
+$body-tertiary-bg: rgba($body-color, .1) !default;
```

Comment:
Can you explain to me what this tertiary-bg thing is? :thinking: Seems weirdthat it is based on body *color* and not *bg-color* anyway no? Maybe a comment should be added to explain too